### PR TITLE
[Process] PhpExecutableFinder: add regression test

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16599 |
| License | MIT |
| Doc PR | - |

I guess it won't be a shame to have this simple test after #16599 .

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
This test doesn't work with the phpdbg runtime.

**Comment by ogizanagi** [comment]
Right. Fixed.

File: src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php [hunk 1]
```diff
@@ -43,7 +43,29 @@ public function testFindWithPhpPath()
...
-     * tests find() with the env var PHP_PATH.
+     * tests find() with the constant PHP_BINARY.
+     */
+    public function testFind()
+    {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Should not be executed in HHVM context.');
+        }
+
+        if (!defined('PHP_BINARY')) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php]: I suggest replacing this check by an `@require php 5.4` annotation

Comment (ogizanagi) [inline src/Symfony/Component/Process/Tests/PhpExecutableFinderTest.php]: Sure :+1:. Thanks.

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**